src/mux-analytics.brs

```diff
@@ -936,6 +949,11 @@ function muxAnalytics() as Object
         m._addEventToQueue(m._createEvent("adplaying"))
       end if
     else if eventType = "PodComplete"
+      if m._lastAdResumeTime <> Invalid
```

Comment:
Not 100% sure, but should you also do this logic on `"Complete"`?

Time between ads in a pod (loading time perhaps potentially) shouldn't be counted.